src/handlers/handlers.rs

```diff
@@ -86,3 +101,29 @@ pub async fn index_page() -> Result<HttpResponse, Error> {
         .content_type("text/html; charset=utf-8")
         .body(html_str))
 }
+
+#[get("/v1/stats/all")]
+async fn status_get_api() -> impl Responder {
+    let sys = System::new();
+
+    match sys.cpu_temp() {
+        Ok(cpu_temp) => println!("\nCPU temp: {}", cpu_temp),
+        Err(x) => println!("\nCPU temp: {}", x),
+    }
+
+    let stats: Stats = get_stats_from_linux(sys).await;
+    println!("{:?}", stats);
+
+    let has_error = stats.cpu_usage == "Error";
+
+    let stats_response = StatsResponse {
+        data: stats,
+        result: !has_error,
```

Comment:
The variables can be assigned values in the order they're defined in the StatsResponse struct, so result would be first and then data.